annotation-processor/src/main/kotlin/pl/touk/krush/source/MappingsGenerator.kt

```diff
@@ -1,28 +1,12 @@
 package pl.touk.krush.source
 
-import com.squareup.kotlinpoet.ClassName
-import com.squareup.kotlinpoet.FileSpec
-import com.squareup.kotlinpoet.FunSpec
-import com.squareup.kotlinpoet.ParameterSpec
+import com.squareup.kotlinpoet.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.